src/lozad.js

```diff
@@ -32,18 +32,17 @@ function markAsLoaded(element) {
   element.setAttribute('data-loaded', true)
 }
 
-const isLoaded = element => element.getAttribute('data-loaded') === 'true'
+const isLoaded = element => element.getAttribute('data-loaded')
```

Comment:
I think you should keep the `=== 'true'` because a `'false'` and a empty string would be evaluated as `true` too.

Reply:
@thiamsantos That is true, wondering if we can't just used a data-loaded attribute then with no value and just check if the element has a data-loaded attribute instead.